tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, pypy, pypy3
+envlist = py26, py27, py33, py34, py35, py36, pypy, pypy3
```

Comment:
You could probably remove py26 and py33, since we don't support those versions anymore.

Reply:
Done.